agent/bench-scripts/pbench-linpack

```diff
@@ -138,21 +194,52 @@ fi
 
 trap "interrupt" INT QUIT TERM
 
-## Run the benchmark and start/stop perf analysis tools
-pbench-start-tools --group=$tool_group --dir=$benchmark_results_dir
+let count=1
+for thread in ${threads//,/ }; do
+	iteration="${count}-${thread}-threads"
+	echo ${iteration} >> ${benchmark_iterations}
```

Comment:
This is done inside `record_iteration`...are we sure we want to do it here as well?

Reply:
Fixed in https://github.com/ldoktor/pbench/pull/5.